<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bidsalyzer</class>
 <widget class="QMainWindow" name="Bidsalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bidsalyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="FileTreeDeselectableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Detected Tasks</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="LocalizerListWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="AnalysisGroupBox_3">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Analysis</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="AnalysisFrame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QCheckBox" name="Eeg2envCheckBox_3">
            <property name="text">
             <string>Generate Analysis File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="AnalysisCcomboBox_3">
            <item>
             <property name="text">
              <string>Hilbert Transform</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="EnvPlotFrame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QCheckBox" name="Env2plotCheckBox_3">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Generate Envellope Plots                           </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="TrialMatFrame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QCheckBox" name="TrialmatCheckBox_3">
            <property name="text">
             <string>Generate Maps (trialmat)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="CorrelationMapFrame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QCheckBox" name="CorrelationMapsCheckBox_3">
            <property name="text">
             <string>Generate Correlation Map</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BrowsePtsButton_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ClearPtsButton_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="StatFileExportFrame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QCheckBox" name="StatFileExportCheckBox_3">
            <property name="text">
             <string>Generate Statistical Export Files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="StatSmoothingComboBox_3">
            <property name="currentText">
             <string extracomment="0">sm0</string>
            </property>
            <item>
             <property name="text">
              <string>sm0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sm250</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sm500</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sm1000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sm2500</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sm5000</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Frequency Bands</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="AllBandsCheckBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Check / Uncheck All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="FrequencyListWidget_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTextBrowser" name="messageDisplayer">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>165</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <layout class="QHBoxLayout" name="BottomHorizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="progressBar_3">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="processButton_3">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Process Analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton_3">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel Analysis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_BIDS_Subject"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionLocalizers"/>
    <addaction name="actionFrequency_Bands"/>
    <addaction name="actionPictures"/>
    <addaction name="actionStatistics"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_BIDS_Subject">
   <property name="text">
    <string>Load BIDS Subject</string>
   </property>
  </action>
  <action name="actionLocalizers">
   <property name="text">
    <string>Localizers</string>
   </property>
  </action>
  <action name="actionFrequency_Bands">
   <property name="text">
    <string>Frequency Bands</string>
   </property>
  </action>
  <action name="actionPictures">
   <property name="text">
    <string>Pictures</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
